noxfile.py

```diff
@@ -68,25 +68,6 @@ def site_packages_dir(session: nox.Session) -> pathlib.Path:
         )
 
 
-def get_circleci_splits(session: nox.Session) -> tuple[int, int]:
```

Comment:
is it now splits by timing?

Reply:
yes it is now splitting by timing.

Since CircleCI will handle the splits now, so we do not need this code.